src/test/java/org/littleshoot/proxy/KeepAliveTest.java

```diff
@@ -144,8 +144,10 @@ public void testProxyKeepsClientConnectionOpenAfterServerDisconnect() throws IOE
             String response = SocketClientUtil.readStringFromSocket(socket);
```

Comment:
I thought perhaps this would be more straightforward, but you're right, it's actually more confusing. I'll use .toLowerCase().